smartsim/_core/_cli/info.py

```diff
@@ -0,0 +1,91 @@
+import importlib.metadata
+import typing as t
+
+from tabulate import tabulate
+
+import smartsim._core._cli.utils as _utils
+import smartsim._core.utils.helpers as _helpers
+from smartsim._core._install.buildenv import BuildEnv as _BuildEnv
+
+if t.TYPE_CHECKING:
+    import argparse
+    import pathlib
+
+_MISSING_DEP = _helpers.colorize("Not Installed", "red")
+
+
+def execute(_args: "argparse.Namespace", /) -> int:
+    print("\nSmart Python Packages:")
+    print(
+        tabulate(
+            [
+                ["SmartSim", _fmt_py_pkg_version("smartsim")],
+                ["SmartRedis", _fmt_py_pkg_version("smartredis")],
+            ],
+            headers=["Name", "Version"],
+            tablefmt="fancy_outline",
+        ),
+        end="\n\n",
+    )
+
+    print("Orchestrator Configuration:")
+    db_path = _utils.get_db_path()
+    db_table = [["Installed", _fmt_installed_db(db_path)]]
+    if db_path:
+        db_table.append(["Location", str(db_path)])
+    print(tabulate(db_table, tablefmt="fancy_outline"), end="\n\n")
+
+    print("Redis AI Configuration:")
+    rai_path = _helpers.redis_install_base().parent / "redisai.so"
+    rai_table = [["Is Installed", _fmt_installed_redis_ai(rai_path)]]
```

Comment:
I'm sure you thought about this for a lot (isn't this the kind of things we spend most of the dev time?) but I don't _love_ the redundancy in `Is installed | Installed` - what about one of the following two:
- `Is installed | Yes/No`
- `Status | Installed/Not installed`

And is there any chance we can also output the Redis AI version?

Reply:
LOL, I also didn't love the redundancy! Changing to `Status`!

Reply:
I couldn't find an immediate way to get the RAI version without saving off some persistent artifact from the build process or without spinning up Redis Server, both of which seemed like a lot of work to try and and cram into a "quick and dirty" implementation for this release.

This is absolutely something that I was planning on tackling this task "properly" in a follow on PR next sprint!